compiler/ilgen/MethodBuilder.cpp

```diff
@@ -139,7 +139,7 @@ MethodBuilder::asMethodBuilder()
 void
 MethodBuilder::initMaps()
    {
-   _parameterSlot = new (PERSISTENT_NEW) TR_HashTabString(typeDictionary()->trMemory());
+   _parameterSlot = std::map<const char *, int32_t>();
```

Comment:
I notice you changed these to all be inline objects in your last commit, so I won't complain about where these all got allocated, other than to (continue to?) hammer on the point that we have to be super careful where these objects get allocated and how their memory is being managed.